python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -398,12 +398,9 @@ def run_transformer_int4_gpu(repo_id,
         tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
         model = model.to('xpu')
     else:
-        if 'starcoder' in repo_id:
-            # Load starcoder-15.5b model in bf16 format to avoid CPU OOM. 
-            model = AutoModelForCausalLM.from_pretrained(model_path, optimize_model=True, load_in_low_bit=low_bit,
-                                                        trust_remote_code=True, use_cache=True, torch_dtype=torch.bfloat16).eval()
-            # Convert the low-bit model back to fp32 for performance considerations.
-            model = model.float()
```

Comment:
comment these instead of delete these? (in case need to add back original model)

Reply:
ok. Submit a new commit in Another PR https://github.com/intel-analytics/BigDL/pull/10315.